new-getting-started-guides.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: 'New Getting Started Guides: Astro, Next.js, Nuxt, Remix, SolidStart, and SvelteKit'
+description: 'Learn how to add Xata database and full-text search functionality to Astro, Next.js, Nuxt, Remix, SolidStart, and SvelteKit with our new Getting Started guides.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/new-getting-started-with-xata-guides.jpg
+  alt: 'Astro, Remix, SolidStart, SvelteKit, Next.js, and Nuxt logos'
+author: Phil Leggetter
+date: 09-01-2023
+published: true
```

Comment:
I'd say let's merge it with publish: false so we can make sure the og images work, and they we switch it to public in a new PR when we're ready for social.

```suggestion
published: false
```